src/utils/worklets/header.cljs

```diff
@@ -0,0 +1,16 @@
+(ns utils.worklets.header)
+
+(def ^:private worklets (js/require "../src/js/worklets/header.js"))
```

Comment:
I think we can give a better name to these files than just header, probably profile-header & profileHeader?